app/views/special_event/HoC2018View.coffee

```diff
@@ -1,8 +1,19 @@
 RootComponent = require 'views/core/RootComponent'
 template = require 'templates/base-flat'
 HoC2018 = require('./HoC2018Component.vue').default
+CreateAccountModal = require 'views/core/CreateAccountModal/CreateAccountModal'
 
 module.exports = class HoC2018View extends RootComponent
   id: 'hoc-2018'
   template: template
   VueComponent: HoC2018
+
+  # Runs before the constructor is called.
+  initialize: ->
```

Comment:
In my recent modal work I moved the initialize code into the constructor and it worked. It may be worth removing this method and moving all `initialize` code into `constructor`.